docroot/modules/custom/webform_serialization/src/Normalizer/WebformNormalizer.php

```diff
@@ -23,6 +24,7 @@ public function getFields($entity, $bundle, ResourceType $resource_type) {
     if (!empty($enabled_public_fields['elements'])) {
       try {
         $parsed = Yaml::decode($enabled_public_fields['elements']);
+        $this->getSelectFields($parsed);
```

Comment:
Typically, when we use `get` to prefix a method, we expect it to return something that we can act on. In this case we're passing in an argument by reference, and acting on it within the body of the method. Let's update the method to be called `populateSelectFieldsWithOptions`, rather than returning the select fields and acting on them here.